From 2bfcdeb09edb871291c6759a83be88b19474b694 Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Thu, 11 May 2017 21:56:13 +0200
Subject: [PATCH 55/67] Gallery2: Increase the size of the tiles when decoding
 images

BitmapRegionDecoder was refactored in Android N [1] and bigger tiles
are now better to render images, so increase the size of each tile.

[1] https://issuetracker.google.com/issues/37136991#comment11

BUGBASH-188
BUGBASH-299

Change-Id: Ie9c282fc5a98c9da6bb8ae3bcfe869b8b422a84b
---
 src/com/android/photos/views/TiledImageRenderer.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/photos/views/TiledImageRenderer.java b/src/com/android/photos/views/TiledImageRenderer.java
index c4e493b34..58bfdbb54 100644
--- a/src/com/android/photos/views/TiledImageRenderer.java
+++ b/src/com/android/photos/views/TiledImageRenderer.java
@@ -159,7 +159,7 @@ public class TiledImageRenderer {
     }
 
     public static int suggestedTileSize(Context context) {
-        return isHighResolution(context) ? 512 : 256;
+        return isHighResolution(context) ? 2048 : 1024;
     }
 
     private static boolean isHighResolution(Context context) {
-- 
2.17.1

